File: java-lhdiff-project/datasets/Aakanksha/File02_Old.java
// File02_Old.java
public class File02 {
    public void processData(String[] data) {
        System.out.println("Starting data processing...");
        for (String item : data) {
            System.out.println("Processing: " + item);
            // Intermediate step 1
            if (item.contains("error")) {
                System.err.println("Error detected for: " + item);
            }
            // Intermediate step 2
        }
        System.out.println("Data processing finished.");
    }
}

File: python-lhdiff-project/datasets/Aakanksha/File02_Old.py
# File02_Old.py

class File02:
    def process_data(self, data):
        print("Starting data processing...")

        for item in data:
            print("Processing:", item)
            # Intermediate step 1
            if "error" in item:
                print("Error detected for:", item, file=sys.stderr)
            # Intermediate step 2

        print("Data processing finished.")